### PR TITLE
development process updates

### PR DESCRIPTION
**Pull request by edublancas**
updated the contribution guidelines based on feedback I've heard from the team

----
:books: Documentation preview :books:: https://ploomber-contributing--82.org.readthedocs.build/en/82/

### CODE REVIEW

File: contributing/process-outline.md
Comment (neelasha23) [inline contributing/process-outline.md:46]: We can mention here that `draft` is when it's still work in progress. Need to mark `ready for review` when work is completed.

File: contributing/review.md
Comment (neelasha23) [inline contributing/review.md:3]: Did you mean `thorough`?

File: contributing/review.md
Comment (neelasha23) [inline contributing/review.md:33]: Maybe contributor can add a comment tagging the reviewer once all comments have been resolved.

File: contributing/submitting-pr.md
Comment (neelasha23) [inline contributing/submitting-pr.md:131]: `implemetned` - typo

File: contributing/submitting-pr.md
Comment (neelasha23) [inline contributing/submitting-pr.md:253]: Screenshot of `convert to draft` link might also help (in case user opens a regular PR)  by mistake

**Comment by neelasha23** [comment]
Added some comments. Do you think it would help if there's a simple flowchart diagram to help understand which tutorial to refer in which scenario? common scenarios maybe `reviewing code` (then check out `Reviewing PRs`, `Maintainer's Guide`..)
@edublancas

File: contributing/submitting-pr.md
Comment (yafimvo) [inline contributing/submitting-pr.md:258]: If you're unable to fix the issues after spending some time on them. You
can message us on Slack so we can help you.

I think it should be one sentence

File: contributing/submitting-pr.md
Comment (yafimvo) [inline contributing/submitting-pr.md:267]: conflicts,  you can -> conflicts, you can (double space)

**Review by yafimvo — CHANGES_REQUESTED** [review]
minor typos

File: contributing/process-outline.md
Comment (edublancas) [reply contributing/process-outline.md:46]: fixed

File: contributing/review.md
Comment (edublancas) [reply contributing/review.md:6]: wdym? in which case?

File: contributing/review.md
Comment (edublancas) [reply contributing/review.md:33]: I think it's better to request another review (added a sentence to be more explicit)

File: contributing/submitting-pr.md
Comment (edublancas) [reply contributing/submitting-pr.md:131]: fixed

File: contributing/submitting-pr.md
Comment (edublancas) [reply contributing/submitting-pr.md:253]: added

File: contributing/process-outline.md
Comment (edublancas) [reply contributing/process-outline.md:15]: fixed

File: contributing/submitting-pr.md
Comment (edublancas) [reply contributing/submitting-pr.md:267]: fixed

**Comment by edublancas** [comment]
thanks for the feedback, addressed all the comments!

> Do you think it would help if there's a simple flowchart diagram to help understand which tutorial

@neelasha23: we have some quick access links in the [home page](https://ploomber-contributing.readthedocs.io/en/latest/intro.html). do you think this can help? Or are you thinking that we can include an actual diagram (perhaps in the outline section?). adding the diagram would be simple with mermaid: https://mermaid.js.org/intro/

File: contributing/review.md
Comment (neelasha23) [reply contributing/review.md:6]: `The only exception is when there's something..` : the above points are also exceptions

File: contributing/submitting-pr.md
Comment (neelasha23) [reply contributing/submitting-pr.md]: I still see `implemetned`

File: contributing/review.md
Comment (edublancas) [reply contributing/review.md:6]: I think the only case when it's fine to "overlook" a first review is when the PR is broken. if more test cases are needed or some refactoring is needed, you can still provide a detailed first review.

File: contributing/submitting-pr.md
Comment (edublancas) [reply contributing/submitting-pr.md]: maybe github  doesn't update the view? this is the commit that I added, you can see I fixed the typo: https://github.com/ploomber/contributing/pull/82/commits/a6895752190de3974ee6cf74d88f1f7ae1de749b

**Comment by edublancas** [comment]
@neelasha23 thanks for the heads up on the "resolve conversation" button. I fixed github permissions (I'll add a little note so people can notify us if they're lacking permissions). please resolve the conversations if you're ok with the changes. I'll open a new issue with the flowchart

File: contributing/review.md
Comment (neelasha23) [reply contributing/review.md:6]: Ok. I think I misunderstood the sentence then . I meant second review doesn't necessarily have to be smaller than first for the above cases. First review can always be the extensive

File: contributing/process-outline.md
Comment (neelasha23) [reply contributing/process-outline.md]: Few typos here:
`opem`, `clicck`

**Review by neelasha23 — APPROVED** [review]
Added a comment on typos. Looks good otherwise!

**Pull request merged**
